ipa-server/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,61 @@
+---
+namespace: ipa
+app_name: ipa
+base_openshift_url: example.com 
+realm: "EXAMPLE.COM"
+
+params:
+  IPA_SERVER_SERVICE: "{{ app_name }}"
+  IPA_SERVER_IMAGE: "freeipa-server:rhel-7"
+  IPA_SERVER_HOSTNAME: "{{ app_name }}.apps.{{ base_openshift_url }}"
+  IPA_SERVER_INSTALL_OPTS: "-U -r {{ realm }} --setup-dns --no-forwarders --no-ntp"
+  IPA_ADMIN_PASSWORD: "Password123"
+  SERVICE_ACCOUNT_USEROOT: "{{ app_name }}-sa"
+  VOLUME_CAPACITY: 1G
+  TIMEOUT: 600
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+  - name: Create Projects
+    template: "https://raw.githubusercontent.com/redhat-cop/openshift-templates/v1.4.4/project-requests/create-project.yml"
+    action: create
+    params_from_vars:
+      NAMESPACE: "{{ namespace }}"
+      NAMESPACE_DISPLAY_NAME: "{{ namespace }}"
+    tags:
+      - project
```

Comment:
Can we prefix these tags with `ipa_`? - i.e.: `ipa_project`, `ipa_imagestream`, etc.

Reply:
Fixed.